cocotb/share/include/gpi.h

```diff
@@ -197,7 +197,8 @@ int gpi_is_indexable(gpi_sim_hdl gpi_hdl);
 // Functions for setting the properties of a handle
 void gpi_set_signal_value_real(gpi_sim_hdl gpi_hdl, double value);
 void gpi_set_signal_value_long(gpi_sim_hdl gpi_hdl, long value);
-void gpi_set_signal_value_str(gpi_sim_hdl gpi_hdl, const char *str);    // String of binary char(s) [1, 0, x, z]
+void gpi_set_signal_value_binstr(gpi_sim_hdl gpi_hdl, const char *str); // String of binary char(s) [1, 0, x, z]
+void gpi_set_signal_value_str(gpi_sim_hdl gpi_hdl, const char *str);    // String of ASCII char(s)
```

Comment:
Can you use `/**  String of ASCII char(s) */` instead, which I think will be picked up by doxygen?

Reply:
That wouldn't be picked up correctly, it must come the line before the declaration and there must be an empty line between declarations. Also, it would be the only thing in the file that has documentation. I think that should be a different commit.

Reply:
Fair